mava/systems/jax/config.py

```diff
@@ -16,6 +16,7 @@
 """Config class for Mava systems"""
 
 from dataclasses import fields, is_dataclass
+from distutils.command.config import config
```

Comment:
This looks like an accidental import? If not, please ignore.

Reply:
Sure is. Great catch :raised_hands: